src/main/java/com/spotify/github/v3/clients/RepositoryClient.java

```diff
@@ -221,7 +221,11 @@ public ChecksClient createChecksApiClient() {
     final String path = String.format(STATUS_URI_TEMPLATE, owner, repo, sha);
     return github
         .post(path, github.json().toJsonUnchecked(request))
-        .thenAccept(IGNORE_RESPONSE_CONSUMER);
+        .thenAccept(response -> {
+          if (response.body() != null) {
+            response.body().close();
+          }
+        });
```

Comment:
what about chaging `IGNORE_RESPONSE_CONSUMER` from `(ignore) -> {}`  to what you have?

Reply:
This is a good idea, I was a little hesitant to make such a change because I only tested it with statuses, but let's be bold:
https://github.com/spotify/github-java-client/pull/70